redis/reply_test.go

```diff
@@ -138,6 +138,11 @@ var replyTests = []struct {
 		ve(redis.Uint64(int64(-1), nil)),
 		ve(uint64(0), redis.ErrNegativeInt(-1)),
 	},
+	{
+		"valueMap([[]byte, []byte])",
+		ve(redis.ValueMap([]interface{}{[]byte("key1"), []byte("v1"), []byte("key2"), []byte("v2")}, nil)),
+		ve(map[string]interface{}{"key1": []byte("v1"), "key2": []byte("v2")}, nil),
```

Comment:
This isn't excessing `interface{}` as all the values are of the type `[]byte` which I don't believe is what the caller will expect.

Reply:
I want to get valueMap . then to individually process each  different types  key , such as  name,age, adult  etc...

`	_, err = c.Do("HMSET", "redis", "name", "go", "age", 18, "adult", true)
	if err != nil {
		fmt.Println(err)
		return
	}
	values, err := redis.ValueMap(c.Do("HGETALL", "redis"))
	if err != nil {
		fmt.Println(err)
		return
	}
	if _, ok := values["name"]; ok {
		value, _ := redis.String(values["name"], err)
		fmt.Printf("%#v\n", value)
	}
	if _, ok := values["age"]; ok {
		value, _ := redis.Int(values["age"], err)
		fmt.Printf("%#v\n", value)
	}
	if _, ok := values["adult"]; ok {
		value, _ := redis.Bool(values["adult"], err)
		fmt.Printf("%#v\n", value)
	}`